app/features/analysis/cards/OneRM/OneRMView.js

```diff
@@ -59,18 +63,82 @@ class OneRM extends Component {
         this.props.changeDays(value);
     }
 
+    _displayTagsToInclude() {
+        if (this.props.tagsToInclude === undefined || this.props.tagsToInclude === null || this.props.tagsToInclude.length === 0) {
+            return (<Text style={[styles.tagText, styles.placeholderText]}>Tags to Include</Text>);
+        }
+
+        var pills = [];
+        let position = 0;
+        this.props.tagsToInclude.map((tag) => {
+            let key = position;
+            pills.push(
+                <Pill key={key} text={tag} style={{paddingRight: 5, paddingBottom: 3, paddingTop: 3}} />
+            );
```

Comment:
- [ ] Instead of pushing into a pills array, just return the pill. and have it be `const pills = this.props.tagsToInclude.map((tag) => {`